docs/providers/openrouter.md

```
@@ -0,0 +1,11 @@
+# OpenRouter Configuration
+
+#### [OpenRouter Integration](https://openrouter.ai/)
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)